pennylane/qnn/torch.py

```diff
@@ -150,6 +150,30 @@ def qnode(inputs, weights_0, weights_1, weights_2, weight_3, weight_4):
 
             qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes, init_method=init_method)
 
+        **Model saving**
+
+        Instances of ``TorchLayer`` can be saved using the usual ``torch.save()`` utility:
+
+        .. code-block::
+
+            qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes)
+            torch.save(qlayer.state_dict(), SAVE_PATH)
+
+        To load the model, an instance of the class must be created first before calling ``torch.load()``,
+        as required by PyTorch:
+
+        .. code-block::
+
+            qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes)
+            qlayer.load_state_dict(torch.load(SAVE_PATH))
+            qlayer.eval()
```

Comment:
```suggestion
            qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes)
            clayer = torch.nn.Linear(2, 2)
            model = torch.nn.Sequential(clayer, qlayer)
            model.load_state_dict(torch.load(SAVE_PATH))
            model.eval()
```
For the comment above.